core/src/test/scala/unit/kafka/coordinator/transaction/TransactionStateManagerTest.scala

```diff
@@ -337,6 +337,62 @@ class TransactionStateManagerTest {
     verifyWritesTxnMarkersInPrepareState(PrepareAbort)
   }
 
+  @Test
+  def shouldRemoveExpiredTransactions(): Unit = {
```

Comment:
scratch that it is possible. I keep forgetting that you don't need to `initPid` for each transaction - doh!